patches/editoria11y--result-name-options-results.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/Dashboard.php b/src/Dashboard.php
+index 2606e9a..00fea21 100644
+--- a/src/Dashboard.php
++++ b/src/Dashboard.php
+@@ -53,7 +53,7 @@ public static function getResultNameOptions(): array {
+
+     $database = \Drupal::database();
+
+-    $result_names = $database->select('editoria11y_dismissals', 't')
++    $result_names = $database->select('editoria11y_results', 't')
```

Comment:
This callback is used as the options callback for both results and dismissals. This seemed like an odd decision for the module to make, since results represents all results, including dismissals. But, the only time a views filter for result names is used in the module's default views reports is for dismissals. I might expand this into a merge request to Editoria11y, but will do so as a follow up.